integration-test/utils/datafactory/schemas/codeofconduct.schema.ts

```diff
@@ -0,0 +1,41 @@
+import { pageSchema } from './page.schema';
+import { linkSchema } from './link.schema';
+
+export const codeofconductSchema = {
+  $ref: '#/definitions/pageSchema',
+  definitions: {
+    codeofconductSchema: {
+      type: 'object',
+      properties: {
+        id: {
+          type: 'string',
+          minLength: 1,
+        },
+        page: {
+          page: { ...pageSchema.definitions.pageSchema },
+        },
+        items: {
+          SectionString: {
+            properties: {
+            title: {
+             type: 'string',
+                minLength: 1,
+              },
+              description: {
+                type: 'string',
+                minLength: 1,
+              },
+              link: { ...linkSchema.definitions.linkSchema },
+              items: {
+                type: 'string',
+                minLength: 1,
+              },
+              additionalProperties: false,
+              required: ['id', 'page', 'items'],
```

Comment:
And here, since `additionalProperties` and `required` are not at the correct level, this test will never fail in case there is no `id` or `page`. I tested it right now. So, these two rules should be on the first level.